config.toml

```diff
@@ -190,11 +190,16 @@ style = "tango"
 # Uncomment if you want your chosen highlight style used for code blocks without a specified language
 # guessSyntax = "true"
 
+[outputFormats.json]
+mediaType = "application/json"
+baseName = "index"
+
 # Everything below this are Site Params
 
 # Comment out if you don't want the "print entire section" link enabled.
-# [outputs]
+[outputs]
 # section = ["HTML", "print", "RSS"]
+home = ["HTML", "json"]
```

Comment:
This will turn off the RSS output format for home. I'm not entirely sure what the RSS output is used for but did you deliberately turn it off?